classes/plugins/ImportExportPlugin.inc.php

```diff
@@ -145,6 +145,26 @@ function getExportFileName($basePath, $objectsFileNamePart, $context, $extension
 		return $basePath . $this->getPluginSettingsPrefix() . '-' . date('Ymd-His') .'-' . $objectsFileNamePart .'-' . $context->getId() . $extension;
 	}
 
+	/**
+	 * Display XML validation errors.
+	 * @param $errors array
+	 * @param $xml string
+	 */
+	function displayXMLValidationErrors($errors, $xml) {
+		echo '<h2>' . __('plugins.importexport.common.validationErrors') .'</h2>';
+
+		foreach ($errors as $error) {
+			switch ($error->level) {
+				case LIBXML_ERR_ERROR:
+				case LIBXML_ERR_FATAL:
+					echo '<p>' .trim($error->message) .'</p>';
+			}
+		}
+		libxml_clear_errors();
+		echo '<h3>' . __('plugins.importexport.common.invalidXML') .'</h3>';
+		echo '<p><pre>' .htmlspecialchars($xml) .'</pre></p>';
```

Comment:
Add whitespace after `.` characters on this line. Also the same a few lines above at `.trim(...`.